File: 2806_account_balance_after_rounded_purchase/Solution.cs
// LeetCode 2806 - Account Balance After Rounded Purchase
// https://leetcode.com/problems/account-balance-after-rounded-purchase/

public class Solution {
    public int AccountBalanceAfterPurchase(int purchaseAmount) {
        int r = ((purchaseAmount + 5) / 10) * 10;
        return 100 - r;
    }
}

File: 2806_account_balance_after_rounded_purchase/solution.php
<?php
// LeetCode 2806 - Account Balance After Rounded Purchase
// https://leetcode.com/problems/account-balance-after-rounded-purchase/

class Solution {
    function accountBalanceAfterPurchase($purchaseAmount) {
        $r = intdiv($purchaseAmount + 5, 10) * 10;
        return 100 - $r;
    }
}
